/**:
  ros__parameters:
    grid_accuracy: [50, 50]
    x_limit: [-1.0, 1.0]
    y_limit: [-1.0, 1.0]
    phi_z_max: 0.5

agent0:
  agent_body:
    ros__parameters:
      init_position: [0.5, 0.0]

agent1:
  agent_body:
    ros__parameters:
      init_position: [0.0, 0.5]

agent2:
  agent_body:
    ros__parameters:
      init_position: [-0.5, 0.5]

agent3:
  agent_body:
    ros__parameters:
      init_position: [0.5, 0.5]

agent4:
  agent_body:
    ros__parameters:
      init_position: [0.5, -0.5]
